pkg/compiler/fuzz/fuzz.go

```diff
@@ -0,0 +1,15 @@
+package compiler_test
+
+import (
+	"github.com/MontFerret/ferret/pkg/compiler"
+)
+
+// This function is our fuzzer
```

Comment:
This comment is not descriptive enough and does not satisfy Go's code style. Write something like
```go
// Fuzz is a fuzzer for oss-fuzz platform.
//
// Read more about it here: https://github.com/google/oss-fuzz
func Fuzz(data []byte) int {}
```